docs-site/docs/assets/dist-train.md

```diff
@@ -0,0 +1,420 @@
+---
```

Comment:
@chrishkchris why is this markdown doc in assets folder?

Reply:
Sorry, I will delete this one (duplicated)

Reply:
Fixed

Reply:
ok!